src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -8233,6 +8233,42 @@ void M()
             await VerifyItemExistsAsync(markup, "Bar");
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ExtensionMethodWithinSameClassOfferedForCompletion()
+        {
+            var markup = @"
+public static class Test
+{
+    static void TestB()
+    {
+        $$
+    }
+    static void TestA(this string s) { }
+}
+";
+            await VerifyItemExistsAsync(markup, "TestA");
+        }
+        
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
```

Comment:
Can you add `WorkItem` attributes to these tests? There are lots of examples in this file.